index.html

```diff
@@ -1,32 +1,164 @@
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+           <div class="card card4">
+            <div class="pic1"><img src="/img/Snapshoot Portfolio (4).png" alt="Snapshoot"></div>
+            <h3 class="project-title">Multi-Post
+              Stories</h3>
+                 <ul class="frame-container">
+                   <li class="frame client">CANOPY</li>
+                   <li class="frame Counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame role">Back End Dev</li>
+                   <li class="frame counter"><img src="/img/Counter.svg" alt=""></li>
+                   <li class="frame year">2015</li>
+                 </ul>
+                 <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="contact-container">
+        <div class="one">
+         <h3 class="about">About Myself</h3>
+         <p class="bio">Hello I’m a software developer!
+             I can help you build a product , feature or website. 
+             Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don’t hesitate 
+             to contact me.
+         </p>
+         <h4 class="connect"><a class="connect-bottom" href="#">LET’S CONNECT</a></h4>
+           <ul class="icon bicon">
+             <li class="github"><a href="#"><img src="/img/github.svg" alt="github"></a></li>   
+             <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt="linkedin"></a></li>
+             <li class="v"><a href="#"><img src="/img/v.svg" alt="v"></a></li>
+             <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt="twiter"></a></li>
+             <li class="m"><a href="#"><img src="/img/m.svg" alt="m"></a></li>
+             
+             
+         </ul>
+         <button class="card-btn resume">Get my resume</button>
+        </div>
+        <div class="two">
+            <div class="title-language">
+                <h2 class="tl">Language</h2>
+                <img src="/img/Icon - Chevron - Down - Larger.svg" alt="" class="dropdow">
+            </div>
+            <ul class="fotter-java">
+              <li class="space"><img src="/img/java.svg" alt="Javascript" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">Javascript</h3></li>
+            </ul>
```

Comment:
- [ ] [OPTIONAL] `ul` is used when you have a collection of items. Here you are incorrectly use the `ul` tag since this is not a list 🔢  Kindly remedy that.

## /!\ This applies to the class **fotter-html**, **fotter-css**, **frameworks-container**, and **skills-container**